### ISSUE TITLE
FileNotFoundError for conda skeleton

### ISSUE THREAD

**Issue by rpanai** [plotly/jupyter-dash#11]
I'm trying to build a conda-forge package via `conda skeleton pypi jupyter_dash` but it returns me the following error

```
Traceback (most recent call last):
  File "setup.py", line 108, in <module>
    with open(os.path.join(here, 'requirements.txt')) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp4lephpktconda_skeleton_jupyter-dash-0.2.1.tar.gz/jupyter-dash-0.2.1/requirements.txt'
$PYTHONPATH = /tmp/tmp4lephpktconda_skeleton_jupyter-dash-0.2.1.tar.gz/jupyter-dash-0.2.1
```

**Comment by jonmmease** [comment]
Hi @rpanai, thanks a lot for looking into getting this on conda-forge!  Looks like we need to include the requirements.txt file in the source package.

I'll have a PR up shortly, and then we can do a release with this change as `0.2.1.post1` if that works for you.

**Comment by jonmmease** [comment]
Fixed in #12 and published to PyPI as `0.2.1.post1`.  The command `conda skeleton pypi jupyter-dash` is completing for me without error now, producing a `jupyter-dash` directory with a meta.yaml file.  Let me know if you run into any other issues. Thanks again!

**Comment by jonmmease** [comment]
I think this has been fixed, so I'm going to close. But we can re-open if you still see an issue on your end. Thanks again for reporting!
